README.md

```diff
@@ -224,7 +224,7 @@ C:\Users\bion\Desktop>dumpbin /IMPORTS .\example.exe | findstr msvcp
 
 1. Follow either [How To Build With A Native Tools Command Prompt][] or [How To Build With The Visual Studio IDE][].
 2. Invoke `git submodule update --init llvm-project` at the root of the STL source tree.
-3. Acquire [Python][] 3.8 or newer.
+3. Acquire [Python][] 3.8 or newer and have it on the `PATH` (or call directly using its full or relative path).

```

Comment:
Minor wording tweaks:

```suggestion
3. Acquire [Python][] 3.8 or newer and have it on the `PATH` (or run it directly using its absolute or relative path).
```

* In filesystem terminology, "absolute" is the opposite of "relative".
* Usually, "run" is for programs while "call" is for functions. Also, I believe that adding "it" here sounds better for this transitive verb.